freeipa/src/main/java/com/sequenceiq/freeipa/flow/freeipa/user/init/FreeIpaClientEventHandlerInitializer.java

```diff
@@ -0,0 +1,54 @@
+package com.sequenceiq.freeipa.flow.freeipa.user.init;
```

Comment:
If we would use `com.sequenceiq.flow.reactor.api.handler.EventHandler` this initializer is not necessary.

Reply:
done